api/src/test/java/org/pmiops/workbench/api/ConceptSetsControllerTest.java

```diff
@@ -263,6 +269,9 @@ WorkbenchConfig workbenchConfig() {
 
   @BeforeEach
   public void setUp() throws Exception {
+    // reset after every test
+    ConceptSetService.MAX_CONCEPTS_PER_SET = 1000; // original value
```

Comment:
needed this, since some test cases set this value to 2 to test over the max case, however the setup that runs before-each test, does not reset this value to the default of 1000.